frontend/src/components/dialogs/SaveLinkDialog.tsx

```
@@ -267,37 +417,109 @@ export function SaveLinkDialog() {
+                        onClick={() => {
+                          if (isPending) {
+                            // pending 폴더 클릭 → 해제하면 AI 추천 취소
+                            setPendingNewFolderName(null);
+                          } else {
+                            // 기존 폴더 클릭 → pending 해제, 기존 폴더 선택
+                            setPendingNewFolderName(null);
+                            setSelectedFolderId(isActive ? null : folder.memberFolderId);
+                          }
+                        }}
+                        className={`${styles.folderTile} ${isActive ? styles.folderTileActive : ''} group`}
+                      >
```

Biome (2.4.6)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)